makefile

```diff
@@ -29,6 +29,9 @@ chrome/script.js: chrome/script.browserify.js
 chrome/inject.js: chrome/inject.browserify.js
 	$(BROWSERIFY) chrome/inject.browserify.js -o chrome/inject.js
 
+chrome/inject_otp.js: chrome/inject_otp.browserify.js
+	$(BROWSERIFY) chrome/inject_otp.browserify.js -o chrome/inject_otp.js
+
```

Comment:
This comment was probably lost in the history: because `inject_otp.js` doesn't have `require()` calls, it doesn't need to be browserified. Just remove this piece, rename the file to `inject_otp.js` and remove this entry from `.gitignore` - I'll take care of doing the same for `inject.browserify.js` separately later today.

Reply:
Sounds good, I was just following what had been done in the past.